core/src/main/scala/package.scala

```diff
@@ -77,7 +77,7 @@ package object quiver {
    * Labeled path through a graph
    * @group types
    */
-  type LPath[N,A] = Adj[N,A]
+  type LPath[N,A] = (N,Vector[(N,A)])
```

Comment:
@kaiserpelagic can you just cast your eye over this to make sure it wont cause problems with nelson?